cmd/init.go

```diff
@@ -108,7 +108,7 @@ func executeInit(namespace, devPath, language, workDir string, overwrite bool) e
 	}
 
 	if checkForDeployment {
-		log.Information("This command walks you through creating an okteto manifest.")
+		log.Information("This command walks you through creating an okteto manifest defining your development container.")
```

Comment:
I'd remove the part of development container unless you explain what the development container is. 
```suggestion
		log.Information("This command walks you through creating an okteto manifest.")
```